docs/HowTo/Get-Started/Connect/Connect-To-Mainnet.md

```diff
@@ -106,6 +106,10 @@ must have the same name as the keys, but use the `.txt` extension.
     If the Launchpad creates a key named `keystore-m_12381_3600_0_0_0-1596485378.json`, then
     the password file must be named `keystore-m_12381_3600_0_0_0-1596485378.txt`.
 
+!!! info
+
+    The password file is [UTF-8 encoded](https://eips.ethereum.org/EIPS/eip-2335#password-requirements).
```

Comment:
The following is just a suggestion:
- do not link `UTF-8` text to EIP-2335 url -> link `EIP-2335` to EIP-2335 URL.
- format is not only about UTF-8, group all the constraints under "EIP-2335 requirements".
- list the requirements, with UTF-8 first to provide an overview of what they are in the EIP.

```suggestion
    The password file format follows [`EIP-2335`](https://eips.ethereum.org/EIPS/eip-2335#password-requirements) requirements (UTF-8 encoded file, unicode normalisation, control code removal)
```